.github/workflows/ci.yaml

```diff
@@ -481,7 +510,7 @@ jobs:
   ready-to-merge:
     name: Ready for merging
     runs-on: ubuntu-20.04
-    needs: deploy
+    needs: [deploy, is-safe-to-merge]
```

Comment:
This one says `is-safe-to-merge` instead of `is-safe-to-run`

Reply:
uuups copilt error